src/main/webapp/release.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+      <title>Release View</title>
+      <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+      <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+     <div class="jumbotron">
+        <h1>SDK Release</h1>      
+    </div>  
+
+    <div class="row">
+        <div class="col col-md-border" >
+            <div class="card text-white">
+                <div class="card">
+                    <h2 class="card-title">M78</h2>
+                    <div class="card-body">
+                        <div class="btn-group">
+                            <a href="enroll.html" class="btn btn-primary" role="button">ENROLL</a>
+                            <button type="button" class="btn btn-info">VIEW</button>
```

Comment:
I think we decided to get rid of these right? Do you want to instead add the release details?